package.json

```diff
@@ -3,11 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "eatlas": "^1.1.0-beta.1",
```

Comment:
Great to have upstream eatlas in there!

Reply:
Yes, and you and SaferActive is contributing to that. So thank you.